src/App.tsx

```diff
@@ -19,13 +19,8 @@ function App() {
     <Router>
       <AuthProvider>
         <NavBar />
+        <NavTrack />
         <Content style={{ padding: '0 50px', margin: '5px 0px', textAlign: 'right' }}>
-          <Breadcrumb style={{ margin: '5px 0' }}>
```

Comment:
- Is this `<Content>` still necessary?
- `NavTrack` is a weird name. Some other options: `NavigationBreadcrumbs` `Breadcrumbs` `NavigationHistory`. Be as explicit as possible with names. Future Gabi will thank you